frontend/src/app/main/ui/workspace/tokens/modals/themes.cljs

```diff
@@ -130,6 +130,7 @@
              [:div {:class (stl/css :theme-row-right)}
               (if-let [sets-count (some-> theme :sets seq count)]
                 [:> button* {:class (stl/css :sets-count-button)
+                             :data-testid "edit-theme"
```

Comment:
This test id can be changed by "aria-labelledby" and an id on the text component down below. 
In the test we can use getbyrole, with the text content as a name.